src/routes/settings/components/Settings.js

```diff
@@ -10,13 +10,13 @@ import type { TFunction } from 'react-i18next'
 import type { NavigationScreenProp } from 'react-navigation'
 import type { SettingsType } from '../../../modules/settings/AppSettings'
 import createSettingsSections from '../createSettingsSections'
-import type { ChangeSettingFunctionType } from '../createSettingsSections'
 import AppSettings, { defaultSettings } from '../../../modules/settings/AppSettings'
 import type { SectionBase } from 'react-native/Libraries/Lists/SectionList'
 
 type PropsType = {|
   theme: ThemeType,
   language: string,
+  city: ?string,
```

Comment:
Probably `cityCode` is better here

Reply:
Yes, kind of. We are really inconsistent with this. E.g. language here is also rather a languageCode than a language.